tests/integration/test_definitions.py

```diff
@@ -327,8 +328,27 @@ def define_tests():
             f"{run} {demo} ; {purge} {demo} -f",
             HasReturnCode(),
         ),
-        "distributed feature_demo": (
-            f"{run} {demo} --vars OUTPUT_PATH=./{OUTPUT_DIR} WORKER_NAME=cli_test_demo_workers ; {workers} {demo} --vars OUTPUT_PATH=./{OUTPUT_DIR} WORKER_NAME=cli_test_demo_workers",
+        "remote feature_demo": (
+            f"{run} {remote_demo} --vars OUTPUT_PATH=./{OUTPUT_DIR} WORKER_NAME=cli_test_demo_workers ; {workers} {remote_demo} --vars OUTPUT_PATH=./{OUTPUT_DIR} WORKER_NAME=cli_test_demo_workers",
+            [
+                HasReturnCode(),
+                ProvenanceYAMLFileHasRegex(
+                    regex="cli_test_demo_workers:",
+                    name="feature_demo",
+                    output_path=OUTPUT_DIR,
+                    provenance_type="expanded",
+                ),
+                StepFileExists(
+                    "verify",
+                    "MERLIN_FINISHED",
+                    "feature_demo",
+                    OUTPUT_DIR,
+                    params=True,
+                ),
+            ],
+        ),
+        "stop workers for distributed feature_demo": (
+            f"{run} {demo} --vars OUTPUT_PATH=./{OUTPUT_DIR} WORKER_NAME=cli_test_demo_workers ; {workers} {demo} --vars OUTPUT_PATH=./{OUTPUT_DIR} WORKER_NAME=cli_test_demo_workers ; sleep 20 ; merlin stop-workers",
```

Comment:
i don't like having this test in here, because it will stop all of the workers the user has connected
i'd be ok with doing it if we did stop-workers --spec {demo}, but that's broken right now

Reply:
Pulled.